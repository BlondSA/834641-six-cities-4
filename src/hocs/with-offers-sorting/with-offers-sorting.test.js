import React from "react";
import renderer from "react-test-renderer";
import withOffersSorting from "./with-offers-sorting.jsx";

const MockComponent = () => {
  return <div></div>;
};

const MockComponentWrapped = withOffersSorting(MockComponent);

describe(`WithOffersSorting snapshot`, () => {
  it(`WithOffersSorting render correctly`, () => {
    const tree = renderer.create(<MockComponentWrapped
      isOpen={true}
      onSortingItemClick={() => {}}
    />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
