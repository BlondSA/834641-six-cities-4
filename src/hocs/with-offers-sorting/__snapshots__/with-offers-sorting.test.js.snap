// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithOffersSorting snapshot WithOffersSorting render correctly 1`] = `<div />`;
